clang/lib/Sema/HLSLExternalSemaSource.cpp

```diff
@@ -131,7 +131,9 @@ struct BuiltinTypeDeclBuilder {
     DeclarationNameInfo NameInfo =
         DeclarationNameInfo(DeclarationName(&II), SourceLocation());
     LookupResult R(S, NameInfo, Sema::LookupOrdinaryName);
-    S.LookupParsedName(R, S.getCurScope(), &SS, false);
+    S.LookupParsedName(R, S.getCurScope(), &SS,
+                       /*ObjectType=*/QualType(),
+                       /*AllowBuiltinCreation*/ false);
```

Comment:
```suggestion
                       /*AllowBuiltinCreation=*/ false);
```

Reply:
Also, why is this specifying 'false' here, which is the default? I realize it was already there, but we should probably not be.

Also-also-- I think this many default arguments has hit the point for ME where we just need an overload for the version where it would be 'all the default things' and specify for everything else.  This is out of hand.

Reply:
@erichkeane I replaced every call to `LookupParsedName` which passed `nullptr`/an empty `CXXScopeSpec` and a null `QualType` with a call to `LookupName`. 

The two parameters with default arguments are forwarded to `LookupQualifiedName`/`LookupName` (the corresponding parameters for those functions also have default arguments).